db/structure.sql

```diff
@@ -1,24 +1,19 @@
 SET statement_timeout = 0;
 SET lock_timeout = 0;
+SET idle_in_transaction_session_timeout = 0;
```

Comment:
about these changes in `structure.sql` I'd say that it's better not to change it, since your features shouldn't change it, because it may have an impact in infra